programs/dynamic-bonding-curve/src/instructions/admin/ix_claim_pool_creation_fee.rs

```diff
@@ -30,24 +27,23 @@ pub struct ClaimCreationFeeCtx<'info> {
 pub fn handle_claim_pool_creation_fee(ctx: Context<ClaimCreationFeeCtx>) -> Result<()> {
     let pool = ctx.accounts.pool.load()?;
 
-    if pool.has_creation_fee() && !pool.creation_fee_claimed() {
+    if pool.has_creation_fee() && !pool.protocol_pool_creation_fee_claimed() {
+        let protocol_pool_creation_fee = pool.get_protocol_pool_creation_fee()?;
         drop(pool);
 
         // Transfer the creation fee to the treasury
-        ctx.accounts
-            .pool
-            .sub_lamports(TOKEN_2022_POOL_WITH_OUTPUT_FEE_COLLECTION_CREATION_FEE)?;
+        ctx.accounts.pool.sub_lamports(protocol_pool_creation_fee)?;
```

Comment:
remove endpoint `handle_withdraw_lamports_from_pool_authority`
creation fee will be sent to pool_authority, and we withdraw from `pool_authority`

Reply:
fixed